src/getHumanAge.test.js

```diff
@@ -7,4 +7,27 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
```

Comment:
this should be wraped up into 'test'

Reply:
dont understand what should be wraped in the test))